src/stories/Overview/Introduction.mdx

```diff
@@ -21,18 +21,32 @@ import packageData from '../../../package.json'
 
 </div>
 
-<div className="card card--company card--shadow-x-small">
-  <div className="card card--company card__info">
-    <img className="card card__company card__info-logo" src="./public/rms-logo.svg" alt="RoleModel Software Logo" />
-    <p>Optics, a RoleModel Software innovation, is a product of our mission to deliver exceptional digital products.</p>
+<div className='flex flex-col gap-md'>
+  <div className="card card--company card--shadow-x-small">
+    <div className="card__info">
+      <img className="card__info-logo" src="./public/rms-logo.svg" alt="RoleModel Software Logo" />
+      Optics, a RoleModel Software innovation, is a product of our mission to deliver exceptional digital products.
+    </div>
+    <a className="btn" href="https://rolemodelsoftware.com/" target="_blank">
+      Learn About RoleModel Software
+      <span className="material-symbols-outlined sb-unstyled">open_in_new</span>
+    </a>
+  </div>
+
+  <div className="card card--company card--shadow-x-small">
+    <div className="card__info">
+      <img className="card__info-logo" src="./public/optics-logo.svg" alt="Optics Logo" />
+      Optics embodies the RoleModel ethos of deep collaboration, uniting our team's diverse expertise with the shared
+      goal of crafting excellent design solutions and our commitment to Character, Collaboration, and Craftsmanship.
+    </div>
+    <a className="btn" href="https://optics.rolemodel.design/" target="_blank">
+      Learn About The Vision Behind Optics
+      <span className="material-symbols-outlined sb-unstyled">open_in_new</span>
+    </a>
   </div>
-  <a className="btn" href="https://rolemodelsoftware.com/">
-    Learn About RoleModel Software
-    <span className="material-symbols-outlined sb-unstyled">open_in_new</span>
```

Comment:
What is `sb-unstyled`?

Reply:
Storybook specific class to remove their default styles that sometimes clash. It's not something that ships with Optics, It only lives on the documentation site.